.github/workflows/test.yml

```diff
@@ -2,9 +2,11 @@ name: Python CI
 
 on:
   pull_request:
+  push:
     branches:
-      - develop
-      - migration/**
+      - main
+      - "release/**"
+      - "feature/**"
```

Comment:
I want `feature/**` to run only when pull_request.
https://github.com/acompany-develop/QuickMPC/pull/31#discussion_r964642482

Reply:
@mdonaka noted with thanks.  I think so.

Reply:
@mdonaka I remote it.  please check :bow:

Reply:
I checked it. Thank you!